src/qt/walletcontroller.cpp

```diff
@@ -2,8 +2,15 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <qt/guiutil.h>
+#include <qt/askpassphrasedialog.h>
+#include <qt/createwalletdialog.h>
+#include <qt/guiconstants.h>
+#include <qt/guiutil.h>
```

Comment:
Travis is not running because of the duplicate `<qt/guiutil.h>` include.

Reply:
Fixed